src/components/Contact.tsx

```diff
@@ -20,8 +21,12 @@ const ContactInfo = styled.p<ContactPageProps>`
 `;
 
 const EmailLink = styled.a<ContactPageProps>`
+  transition: 2s;
   &:hover {
-    text-decoration: underline;
+    ::before,
+    ::after {
+      content: "${" "}+${" "}";
```

Comment:
![image](https://user-images.githubusercontent.com/11725595/104601486-97efb500-567a-11eb-806b-4323ff55bdb3.png)

is that the desired outcome? I dont have a 2s transition since before the hover there is no `content` value set. `transition` works when a value is going "from" an old value "to" a new value

Reply:
leftover code, thanks for spotting

Reply:
good definition btw